workflows-scripts/inventory/item-load-inventory/README.md

```diff
@@ -0,0 +1,33 @@
+## Workflow steps:
+### Pre-requisite:
+1. Open Inventory App
+2. Select Instance tab in top left
+3. Select Keyword(title, contributor, identifier) in the drop down and type "aba" in search field
+
+
+## Workflow steps:
+##### Workflow for inventory_loadItems.jmx
+1. Select the third instance "ABA 7th National Conference on Law Office Economics 
+and Management[sponsored by] Section of Economics of Law Practice, American Bar Association."
+2. Check that items from Holding section have "Available" or "In transit" status 
+3. Copy one of the item barcodes in the Holding section
+4. Go to Checkout page
+5. Open "Patron look-up" modal window
+6. Select an active user with a barcode and close "Patron look-up" modal window
+7. Put the item barcode to "Scan or enter item barcode" field
+8. Click "Enter" button 
+9. Return to "Inventory app"
+10. Wait until all items records appear in the Holdings section
+
```

Comment:
Can you describe steps, for example, SQL queries on how did you generate JMeter supported test data?

Reply:
Added